build.gradle

```diff
@@ -25,17 +33,78 @@ repositories {
         name = 'Glass Releases'
         url = 'https://maven.glass-launcher.net/releases'
     }
+    // Used for StationAPI and HowManyItems.
+    maven {
+        name = 'Glass Snapshots'
+        url = 'https://maven.glass-launcher.net/snapshots'
+    }
+    // Used for a StationAPI dependency.
+    maven {
+        name = 'Froge'
+        url 'https://maven.minecraftforge.net/'
+    }
+    // Used for projects that do not have a maven repository, but do have a GitHub repository with working build scripts.
+    maven {
+        name = 'Jitpack'
+        url = 'https://jitpack.io'
+    }
+    // Used for another StationAPI dependency
+    exclusiveContent {
+        forRepository {
+            maven {
+                name = "Modrinth"
+                url = "https://api.modrinth.com/maven"
+            }
+        }
+        filter {
+            includeGroup "maven.modrinth"
+        }
+    }
+    mavenCentral()
 }
 
-
-
 dependencies {
+    // To change the versions see the gradle.properties file
     minecraft "com.mojang:minecraft:${project.minecraft_version}"
     mappings "net.glasslauncher:biny:${project.yarn_mappings}:v2"
     modImplementation "babric:fabric-loader:${project.loader_version}"
 
     implementation "org.slf4j:slf4j-api:1.8.0-beta4"
-    implementation "org.apache.logging.log4j:log4j-slf4j18-impl:2.16.0"
+    implementation 'org.apache.logging.log4j:log4j-slf4j18-impl:2.17.2'
+    implementation "blue.endless:jankson:1.2.1"
+    implementation(include("me.carleslc:Simple-Yaml:1.8.4") as Dependency)
+    implementation("io.github.llamalad7:mixinextras-fabric:${project.properties["mixinextras_version"]}")
+
+    modImplementation "net.modificationstation:StationAPI:${project.stapi_version}"
+
+    // Optional, but convenient mods for mod creators and users alike.
+    modImplementation("net.glasslauncher.mods:ModMenu:${project.modmenu_version}") {
+        transitive false
+    }
+
+    modImplementation("net.glasslauncher.mods:glass-networking:${project.glass_networking_version}") {
+        transitive false
+    }
+
+    modImplementation("net.glasslauncher.mods:GlassConfigAPI:${project.gcapi_version}") {
+        transitive false
+    }
+
+    modImplementation("net.glasslauncher.mods:AlwaysMoreItems:${project.alwaysmoreitems_version}") {
+        transitive false
+    }
+
+    modImplementation("com.github.matthewperiut:retrocommands:${project.retrocommands_version}") {
+        transitive false
+    }
+
+    modImplementation("net.danygames2014:spawneggs:${project.spawneggs_version}") {
+        transitive false
+    }
```

Comment:
unneeded?

Reply:
Probably not needed for this mod yeah, I can remove it after work or you can now

Reply:
Removed it via phone commit